spec/cassette/client/cache_spec.rb

```diff
@@ -1,17 +1,13 @@
-# encoding: utf-8
-
-
+# frozen_string_literal: true
 
 describe Cassette::Client::Cache do
   it 'uses the cache store set in configuration' do
     # setup
-    global_cache = double('cache_store')
+    global_cache = instance_double(described_class, 'cache_store')
```

Comment:
If you want to be thorough, global cache is a [ActiveSupport::Cache::Store](https://api.rubyonrails.org/classes/ActiveSupport/Cache/Store.html) and not a Cassette::Client::Cache